test/check-img.js

```diff
@@ -0,0 +1,22 @@
+import fs from 'fs';
+import path from 'path';
+
+import {assert} from 'chai';
+
+import {languages} from '../lib/languages';
+
+const img_dir = path.resolve('views/resources/logos');
+
+function checkImage(lang) {
+    let result = fs.existsSync(img_dir + '/' + lang.logoUrl);
+    if (lang.logoUrlDark !== null) {
+        result = result && fs.existsSync(img_dir + '/' + lang.logoUrlDark);
+    }
+    return result;
+}
+
+describe('Image-checks', () => {
```

Comment:
I _love_ this! Thank you so much!

Using `fs-extra` instead of `fs`, one could change this to only `readdir` in the `before` once and check its output agianst each language icon. But it's not worth the hassle.

Reply:
Yeah I'm not a JS dev, so I tried to do this:
https://github.com/compiler-explorer/compiler-explorer/blob/a4565fb352b724ee9eddde4218326c2d28352820/static/main.js#L56
But then learned its a webpack thing.